src/app/components/client/ExposureCard.tsx

```diff
@@ -321,15 +322,9 @@ const SubscriberBreachCard = (props: SubscriberBreachCardProps) => {
       <div className={styles.detailsFoundItem}>
         <dt>
           <span className={styles.exposureTypeIcon}>{props.icon}</span>
-          {props.exposureCategoryLabel}
+          {props.exposureCategoryLabel}: {props.count}
```

Comment:
@flodolo Does this work? Or do we need to let localizers control the order of `label: count`?

Reply:
It's not the order, it's the spaces. This needs to be exposed as a localizable string.